api/provider/aws/environment_create.go

```diff
@@ -162,7 +169,13 @@ func (e *EnvironmentProvider) createLC(
 	return nil
 }
 
-func (e *EnvironmentProvider) createASG(autoScalingGroupName, launchConfigName string, minSize int64, privateSubnets []string) error {
+func (e *EnvironmentProvider) createASG(
+	autoScalingGroupName string,
```

Comment:
[mega nit] `autoScalingGroupName, launchConfigName string,.....`

Reply:
when we need to line wrap we put each param name and type on it's own line

Reply:
TIL!